client/landing/jetpack-cloud/components/current-site/style.scss

```diff
@@ -0,0 +1,46 @@
+.current-site.card {
+	margin: 0;
+	padding: 0;
+	box-shadow: none;
+	background: transparent;
+
+	&.is-loading {
+		.site-icon {
+			animation: pulse-light 0.8s ease-in-out infinite;
+		}
+
+		.site__title {
+			color: var( --color-neutral-50 );
```

Comment:
Any reason why we don't want to use a theme-specific variable here, e.g. `--color-sidebar-text`? We're already using `--color-sidebar-background` and `--color-sidebar-border` in the same file so I guess it would make sense.